service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceRegisterTask.java

```diff
@@ -145,7 +145,7 @@ private boolean registerSchemas() {
       String schemaId = entry.getKey();
       String content = entry.getValue();
       GetSchemaResponse existSchema = extractSchema(schemaId, existSchemas);
-      boolean exists = existSchema != null;
+      boolean exists = existSchema != null && existSchema.getSummary() != null;
```

Comment:
SC support batch register schemas
we should swtich to new api

be careful that maybe will send too big request, need to split to multi request by max size of limit.